api/tests/devcomp/unit/domain/models/element/QROCM-for-answer-verification_test.js

```diff
@@ -51,11 +51,11 @@ describe('Unit | Devcomp | Domain | Models | Element | QrocMForAnswerVerificatio
       });
 
       // then
-      expect(qrocm.solutions).deep.equal({
+      expect(qrocm.solutionValue).deep.equal({
```

Comment:
Ce changement de propriété est il à répercuter sur les QCU pour généraliser ?

Reply:
yes why not

Reply:
décision documentée dans [ce commentaire](https://github.com/1024pix/pix/pull/7984#discussion_r1481443816).